tests/test_iterator.rs

```diff
@@ -170,8 +170,8 @@ pub fn test_iterator() {
     iter.next();
     assert!(!iter.valid());
     // Once iterator is invalid, it can't be reverted.
-    iter.prev();
-    assert!(!iter.valid());
+    //iter.prev();
```

Comment:
Why comment out this two lines?

Reply:
Because when iterator is not valid anymore, `prev` should not be called, otherwise assertion in prev will fail.